card/mod/standard/set/all/rich_html/related.rb

```diff
@@ -0,0 +1,31 @@
+format :html do
+
+  # the related view nests a related card with a submenu
+  # the subcard is specified as an "item" card using the slot/voo api
+  view :related, cache: :never do
+    name, options = related_item
+    return unless related_card name
+    voo.show :toolbar, :menu, :help
+    frame do
+      voo.hide :header, :toggle
+      nest @related_card, related_options(options)
+    end
+  end
+
+  def related_item
+    options = voo.items
+    voo.items = {}
```

Comment:
this line (`voo.items = {}`) is the difference.  we don't want the configuration of a specific related view to get stored in the slot, because it interferes with all the other related views.